src/core/Surface/Surface.tsx

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
```

Comment:
Wondering, if we should consider having folder for these `helper` components...

Reply:
Also, another idea :D
Would it make sense to have a hook, which would just deliver a class or style, so you could apply that on already existing components?
I would add that to popover, for example (instead of reusing mixing in css)